event_response_tool/event_response.py

```diff
@@ -288,34 +474,63 @@ def construct_argument_parser():
     username = args.username
     password = args.password
 
-    # Mandatory event weight argument
+    # Event weight CSV
     event_weights_csv = args.event_weights_csv
 
-    # LayerView manipulation arguments
-    layer_views_csv = args.layer_views_csv
-    portfolio_view_uuid = args.portfolio_view_uuid
+    # Method to be used for Event Response
+    mixture_distribution_method = (
+        True if args.method == "mixture_distribution" else False
```

Comment:
Does this align with the boolean flag used in the notebook? I might be missing where that gets converted

Reply:
Yes,  this boolean flag aligns with the one used in the notebook. So argparse has something called choices which restrict user input to be a specific value. In our case, I am restricting the user to choose between "mixture_distribution" and "convolution"  for the --method command line argument (this is done in `construct_argument_parser()` function). Then while parsing the arguments, I am setting the `mixture_distribution_method` flag's boolean value based on the user input for --method.